packages/design-system/src/components/tooltip/index.js

```diff
@@ -267,6 +267,7 @@ function Tooltip({
         spacing={spacing}
         isOpen={Boolean(shown && (shortcut || title))}
         onPositionUpdate={positionArrow}
+        noOverFlow
```

Comment:
Added this because of `overflow-y:auto` used to display scrollbars on tooltip and made the UI look bad, therefore if `noOverFlow` is passed then `overflow-y` will not be attached to the popup.

Reply:
Yea it seems it is now forcing the scroll bar on `ColorPicker` to outside of component. 
|Main|This pr|
|--|--|
|![main scroll](https://user-images.githubusercontent.com/1820266/151440015-ae20cf67-9ac9-412a-b7bb-57f061eafa57.gif)|![this pr scroll bar](https://user-images.githubusercontent.com/1820266/151439733-8e75d953-bbfe-49ce-98f1-8a9f20b53b7c.gif)|